org.leeroymanea.synology-note-station.json

```diff
@@ -0,0 +1,45 @@
+{
+  "app-id": "org.leeroymanea.synology-note-station",
```

Comment:
the domain `leeroymanea.org` does not resolve

Reply:
also this does not look like the upstream developers domain

Reply:
Other synology package https://github.com/flathub/com.synology.SynologyDrive

So it should build on the same domain.